src/main/java/sx/blah/discord/api/internal/Requests.java

```diff
@@ -56,62 +58,67 @@
 	 */
 	public static final Requests GENERAL_REQUESTS = new Requests(null);
 
+	/**
+	 * The HTTP client requests are made on.
+	 */
+	private final OkHttpClient HTTP;
```

Comment:
Can this as well as the DiscordRequest fields be renamed to follow convention? (I know you didn't name the other ones)